tests/depth_dependent_box_function_time_dependent.prm

```diff
@@ -0,0 +1,175 @@
+# Test depth-dependent material model using simple material model as
+# base model with depth-dependence specified by user-provided function
```

Comment:
Can you say that this tests the time dependence of the function in the depth dependent material model?